### PR TITLE
RavenDB-11316 Embed LE authority certificate into Linux docker image

### PR DESCRIPTION
**Pull request by gregolsky**

### CODE REVIEW

File: docker/ravendb-ubuntu1604/Dockerfile [hunk 1]
```diff
@@ -9,14 +8,17 @@ ENV RAVEN_ARGS='' RAVEN_SETTINGS='' RAVEN_Setup_Mode='Initial' RAVEN_DataDir='Ra
 
 EXPOSE 8080 38888 161
 
+ADD https://letsencrypt.org/certs/lets-encrypt-x3-cross-signed.pem.txt /usr/share/ca-certificates/lets-encrypt-x3-cross-signed.pem
```
Comment (ayende) [inline docker/ravendb-ubuntu1604/Dockerfile:11]: Don't we nee this one ? 

http://apps.identrust.com/roots/dstrootcax3.p7c
--

Comment (gregolsky) [reply docker/ravendb-ubuntu1604/Dockerfile:11]: That certificate is already there. I think we lacked intermediate one locally.
See http://issues.hibernatingrhinos.com/issue/RavenDB-11316#comment=67-29057

```
Updating certificates in /etc/ssl/certs...
1 added, 0 removed; done.
```

Comment (ayende) [reply docker/ravendb-ubuntu1604/Dockerfile:11]: Did you check the remote calls there?

Comment (gregolsky) [reply docker/ravendb-ubuntu1604/Dockerfile:11]: Was busy fixing release, will do that today.

**Comment by gregolsky** [comment]
I'll reopen once I confirm it's fine.

**Pull request closed without merging**